master/docs/react/practical-projects.zh-CN.md 项目实战文档缺少部分步骤

- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://ant-design.gitee.io/docs/react/practical-projects-cn](https://ant-design.gitee.io/docs/react/practical-projects-cn)

### Steps to reproduce
按照步骤(1)安装 Umi(2)新建路由(3)编写 UI Component(4)简单数据流方案 新建文件和代码 执行yarn start 后编译报错


### What is expected?
编译通过，访问 http://localhost:8000/products 链接能看到文档内展示的效果

### What is actually happening?
编译报错：
Failed to compile
./src/models/useProductList.ts
Module not found: Can't resolve '@/services/product' in '\myapp\src\model

| Environment | Info |
|---|---|
| antd | 4.18.5 |
| React | 17.x |
| System | windows 10 pro |
| Browser | Google Chrome  Version 83.0.4103.97  |

---
文档中第四部分简单数据流方案引用的文件在文档中没有说明，项目文件夹内没有该文件所以报错
import { queryProductList } from '@/services/product';


<!-- generated by ant-design-issue-helper. DO NOT REMOVE -->

--- a/docs/react/practical-projects.en-US.md
+++ b/docs/react/practical-projects.en-US.md
@@ -115,15 +115,11 @@ export function queryProductList() {
       resolve([
         {
           id: 1,
-          name: 'product 1',
+          name: 'dva',
         },
         {
           id: 2,
-          name: 'product 2',
-        },
-        {
-          id: 3,
-          name: 'product 3',
+          name: 'antd',
         },
       ]);
     }, 2000);

--- a/docs/react/practical-projects.en-US.md
+++ b/docs/react/practical-projects.en-US.md
@@ -100,7 +100,38 @@ export default ProductList;
 
 The file name corresponds to the name of the final model, and you can consume the data in the model through the API provided by the plug-in.
 
-Let's take a simple table as an example. First you need to create a new file `src/models/useProductList.ts`.
+Let's take a simple table as an example. First we create a new file `src/services/product.ts` for remote API.
+
+```tsx
+/*
+export function queryProductList() {
+  return fetch('/api/products').then(res => res.json());
+}
+*/
+// mock request service by setTimeout
+export function queryProductList() {
+  return new Promise(resolve => {
+    setTimeout(() => {
+      resolve([
+        {
+          id: 1,
+          name: 'product 1',
+        },
+        {
+          id: 2,
+          name: 'product 2',
+        },
+        {
+          id: 3,
+          name: 'product 3',
+        },
+      ]);
+    }, 2000);
+  });
+}
+```
+
+Then you need to create a new file `src/models/useProductList.ts`.
 
 ```tsx
 import { useRequest } from 'umi';

--- a/docs/react/practical-projects.zh-CN.md
+++ b/docs/react/practical-projects.zh-CN.md
@@ -112,15 +112,11 @@ export function queryProductList() {
       resolve([
         {
           id: 1,
-          name: 'product 1',
+          name: 'dva',
         },
         {
           id: 2,
-          name: 'product 2',
-        },
-        {
-          id: 3,
-          name: 'product 3',
+          name: 'antd',
         },
       ]);
     }, 2000);

--- a/docs/react/practical-projects.zh-CN.md
+++ b/docs/react/practical-projects.zh-CN.md
@@ -97,7 +97,38 @@ export default ProductList;
 
 文件名则对应最终 model 的 name，你可以通过插件提供的 API 来消费 model 中的数据。
 
-我们以一个简单的表格作为示例。首先需要新建文件 `src/models/useProductList.ts`。
+我们以一个简单的表格作为示例。首先新建一个 `src/services/product.ts` 存放产品相关的接口。
+
+```tsx
+/*
+export function queryProductList() {
+  return fetch('/api/products').then(res => res.json());
+}
+*/
+// 先用 setTimeout 模拟一个请求，正常的写法如上所示
+export function queryProductList() {
+  return new Promise(resolve => {
+    setTimeout(() => {
+      resolve([
+        {
+          id: 1,
+          name: 'product 1',
+        },
+        {
+          id: 2,
+          name: 'product 2',
+        },
+        {
+          id: 3,
+          name: 'product 3',
+        },
+      ]);
+    }, 2000);
+  });
+}
+```
+
+然后新建文件 `src/models/useProductList.ts`。
 
 ```tsx
 import { useRequest } from 'umi';

